frontend/public/components/nav/perspective-nav.tsx

```diff
@@ -49,7 +49,7 @@ const PerspectiveNav: React.FC<{}> = () => {
         idx={idx}
         resourceRef={resource}
         onChange={setPinnedResources}
-        onDrag={setValidPinnedResources}
+        onDrag={setPinnedResources}
```

Comment:
@jeff-phillips-18 The intent was not to update the user-settings till the dragged resource is dropped on the target area

Reply:
PTAL https://github.com/openshift/console/pull/11266